packages/new-design/shared/lab/components/header.js

```diff
@@ -66,8 +66,10 @@ const Header = ({ app }) => {
                 <span className="ml-2">{t('docs')}</span>
               </a>
               <Link href="/support" role="button" className={btnClasses}>
-                <HelpIcon />
-                <span className="ml-2">{t('support')}</span>
+                <span>
```

Comment:
This is great, thank you.

But, since we're fixing it, we should use an empty fragment here as that would fix the issue without introducing additional markup.

Like this:
```
<Link href="/support" role="button" className={btnClasses}>
  <>
    <HelpIcon />
    <span className="ml-2">{t('support')}</span>
  </>
</Link>
```

See here for more info about fragments in React: https://reactjs.org/docs/fragments.html

Reply:
Will change it!